snapshot/android/snapshot-generator.js

```diff
@@ -217,3 +244,134 @@ SnapshotGenerator.prototype.generate = function (options) {
         return this.buildPath;
     });
 }
+
+SnapshotGenerator.prototype.getSnapshotToolCommand = function (snapshotToolPath, inputFilePath, outputPath, toolParams) {
+    return `${snapshotToolPath} ${inputFilePath} --startup_blob ${join(outputPath, `${SNAPSHOT_BLOB_NAME}.blob`)} ${toolParams}`;
+}
+
+SnapshotGenerator.prototype.getXxdCommand = function (srcOutputDir) {
+    return `xxd -i ${SNAPSHOT_BLOB_NAME}.blob > ${join(srcOutputDir, `${SNAPSHOT_BLOB_NAME}.c`)}`;
+}
+
+SnapshotGenerator.prototype.getPathInDocker = function (mappedLocalDir, mappedDockerDir, targetPath) {
+    if (!isSubPath(mappedLocalDir, targetPath)) {
+        throw new Error(`Cannot determine a docker path. '${targetPath}' should be inside '${mappedLocalDir}'`)
+    }
+
+    const pathInDocker = join(mappedDockerDir, relative(mappedLocalDir, targetPath));
+
+    return convertToUnixPath(pathInDocker);
+}
+
+SnapshotGenerator.prototype.handleSnapshotToolResult = function (error, stdout, stderr, androidArch) {
```

Comment:
`inputFile` is not provided as argument to this function:

```suggestion
SnapshotGenerator.prototype.handleSnapshotToolResult = function (inputFile , error, stdout, stderr, androidArch) {
```